extension/src/XLIFFDocument.ts

```diff
@@ -147,7 +147,7 @@ export class Xliff implements XliffDocumentInterface {
     public translationMap(): Map<string, string[]> {
         let transMap = new Map<string, string[]>();
         this.transunit.filter(t => !isNullOrUndefined(t.target) && t.targetsHasTextContent()).forEach(unit => {
-            if (!isNullOrUndefined(unit.target)) {
+            if (!isNullOrUndefined(unit.target) && unit.target.length > 0) {
```

Comment:
This could be removed completely since targetsHasTextContent() could not return true if there are no targets... but I've added hasTargets and are using that in other places.